modules/client/examples/03-tokenVoting-client/11-get-members.ts

```diff
@@ -30,11 +34,25 @@ Returns:
 
 ```json
 [
```

Comment:
Let's also add a vanilla example, of someone not delegating at all (null delegatee, etc)